databases/orientdb/src/main/java/com/gentics/mesh/graphdb/cluster/TopologyEventBridge.java

```diff
@@ -113,14 +113,16 @@ public void onDatabaseChangeStatus(String nodeName, String iDatabaseName, DB_STA
 		}
 		databaseStatusMap.put(nodeName, iNewStatus);
 		log.info("Node {" + nodeName + "} Database {" + iDatabaseName + "} changed status {" + iNewStatus.name() + "}");
+		boolean isMe = "storage".equals(iDatabaseName) && nodeName.equals(manager.getNodeName());
```

Comment:
`isMe` will be true if the current node in a cluster is the manager, right?
I suggest renaming it to `isManager`

Reply:
`manager.getNodeName()` returns the name of a *current* node:
```
	/**
	 * Determine the OrientDB Node name.
	 * 
	 * @return
	 */
	public String getNodeName() {
		StringBuilder nameBuilder = new StringBuilder();
		String nodeName = options.getNodeName();
		nameBuilder.append(nodeName);

		// Sanitize the name
		String name = nameBuilder.toString();
		name = name.replaceAll(" ", "_");
		name = name.replaceAll("\\.", "-");
		return name;
	}
```
So whenever the manager marks the current node as ready, the initial Mesh node starting process is unlocked.